test/testDrivenRange.test.js

```diff
@@ -0,0 +1,44 @@
+const alerts = require('../testDrivenRanges');
+const {expect} = require('chai');
+const sinon = require('sinon');
+
+describe('sortArrayOfNumbers', function() {  
+ it('has function sortArrayOfNumbers', ()=>{
+    expect(alerts.sortArrayOfNumbers([6,9,7])).to.exist;    
+ })
+
+ it('Check the sorting',()=>{
+  let x= [6,9,7];
+  let res = alerts.sortArrayOfNumbers(x);
+  expect(res).to.be.deep.equal([6,7,9]);
+ })
+
+ it('check function rangeReading', ()=>{
+  let input = [3, 5,3, 4, 10, 11, 12,10];
+  let actual = alerts.rangeReading(input);
+  let output = [ [ 'Range', 'Readings' ], [ 3, '-', 5, 5 ], [ 10, '-', 12, 5 ] ];
+  expect(actual).to.deep.equal(output);    
+})
+
+it('check function printRangeAndReadings', ()=>{
+  let input = [ [ 3, 3, 4, 5 ], [ 10, 10, 11, 12 ] ];
+  let actual = alerts.rangeReading(input);
+  console.log(actual);
+  let output = [
+    [ 'Range', 'Readings' ],
+    [ [ 3, 3, 4, 5 ], '-', [ 3, 3, 4, 5 ], 2 ],       
```

Comment:
Is this the required output as defined in the requirement?
The expected output would be 3-5,4   10-12,4
Please correct me if wrong...

Reply:
Has this comment been addressed?